src/main/java/io/github/moulberry/notenoughupdates/NEUManager.java

```diff
@@ -430,6 +430,33 @@ private String searchDebug(String[] searchArray, ArrayList<DebugMatch> debugMatc
 		return debugBuilder.toString();
 	}
 
+	/**
+	 * SearchString but with AND | OR support
+	 */
+
+	public boolean MultiSearchString(String match, String query) {
```

Comment:
mULTIsEARCHsTRING

Reply:
first thing I thought of, if you got a better idea, then please suggest so.

Reply:
multiSearchString 🤓

Reply:
> multiSearchString nerd_face

hmmm, I don't think I can put a space in a method name :trollface:

Reply:
I was thinking about `multiSeachString`. It took me 5 seconds to realize you dont have an inner class with constructor there but a normal method.
Please use the camel case convention. See https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

Reply:
ah yeah, fair, don't know how I forgot that